packages/tools/client-debugger/client-debugger-view/src/components/AudienceStateTable.tsx

```diff
@@ -47,8 +52,18 @@ export function AudienceStateTable(props: AudienceStateTableProps): React.ReactE
 				<TableRow>
 					{audienceStateColumns.map((column, columnIndex) => (
 						<TableHeaderCell key={columnIndex}>
-							{column.columnKey === "clientId" && <Person24Regular />}
-							{column.columnKey === "userId" && <Person24Regular />}
+							<TooltipHost
+								content="Represents the connection between Fluid Runtime and the Fluid server."
```

Comment:
Nit: "service" instead of "server" maybe? We could also explicitly call out the sequencing service here, since that's ultimately what we're referring to.

Reply:
I'd even rephrase it like `Id assigned by the Fluid [sequencing] service to the current connection. It will change if the client disconnects and reconnects.`. "Represents the connection" feels slightly weird in ways I can't quite pinpoint.

Reply:
`ID assigned by the Fluid (sequencing) service to the current connection. Subject to change if the client disconnects or reconnects.`

Change to disconnects or reconnects, as Fluid service might generate new `clientID` after certain amount of time even if the user did not intend to disconnect from the server.

Reply:
> Change to disconnects or reconnects, as Fluid service might generate new clientID after certain amount of time even if the user did not intend to disconnect from the server.

Oh really? Interesting, good to know.

Reply:
Bringing @tyler-cai-microsoft for more details 😉.